packages-new/components/file/stencil.config.ts

```diff
@@ -0,0 +1,29 @@
+import { Config } from '@stencil/core';
+import { getStencilConfig } from '@ovhcloud/ods-common-stencil';
+import * as jestConfig from './jest.config';
+
+const args = process.argv.slice(2);
+
+export const config: Config = getStencilConfig({
+  namespace: 'osds-file',
+  args,
+  jestConfig: jestConfig.default,
+  reactOutput: {
+    componentCorePackage: '@ovhcloud/ods-component-file',
+    // exclude peer dependencies that corresponds to www usage
+    excludeComponents: ['osds-icon', 'osds-link', 'osds-text']
```

Comment:
regarding component dependencies, should be
`['osds-button', 'osds-icon', 'osds-link', 'osds-progress-bar']`